src/Controller/Admin/AdminPageController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Repository\UserRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+class AdminPageController extends AbstractController
```

Comment:
poti sa scoti Admin din nume, e deja in namespace

Reply:
ok.